src/org/thoughtcrime/securesms/components/ThumbnailView.java

```diff
@@ -293,11 +296,15 @@ protected void onPostExecute(Boolean success) {
   public ListenableFuture<Boolean> setImageResource(@NonNull GlideRequests glideRequests, @NonNull Uri uri) {
     SettableFuture<Boolean> future = new SettableFuture<>();
 
-    glideRequests.load(new DecryptableUri(uri))
-                 .diskCacheStrategy(DiskCacheStrategy.NONE)
-                 .transforms(new CenterCrop(), new RoundedCorners(radius))
-                 .transition(withCrossFade())
-                 .into(new GlideDrawableListeningTarget(image, future));
+    GlideRequest request = glideRequests.load(new DecryptableUri(uri))
+                                        .diskCacheStrategy(DiskCacheStrategy.NONE);
+    if (radius > 0) {
+      request = request.transforms(fit, new RoundedCorners(radius));
+    } else {
+      request = request.transforms(fit);
+    }
+    request.transition(withCrossFade())
+	   .into(new GlideDrawableListeningTarget(image, future));
```

Comment:
This function is unused, you can just delete it to resolve the merge conflict :)